packages/@angular/cli/models/config.ts

```diff
@@ -52,6 +52,17 @@ export class CliConfig extends CliConfigBase<ConfigInterface> {
       cliConfig.alias('apps.0.prefix', 'defaults.prefix')
     ];
 
+    // Additional aliases which do not emit any messages.
+    cliConfig.alias('defaults.interface.prefix', 'defaults.inline.prefixInterfaces');
+    cliConfig.alias('defaults.component.inlineStyle', 'defaults.inline.style');
+    cliConfig.alias('defaults.component.inlineTemplate', 'defaults.inline.template');
+    cliConfig.alias('defaults.component.spec', 'defaults.spec.component');
+    cliConfig.alias('defaults.class.spec', 'defaults.spec.class');
+    cliConfig.alias('defaults.component.directive', 'defaults.spec.directive');
+    cliConfig.alias('defaults.component.module', 'defaults.spec.module');
+    cliConfig.alias('defaults.component.pipe', 'defaults.spec.pipe');
+    cliConfig.alias('defaults.component.service', 'defaults.spec.service');
+
     // If any of them returned true, output a deprecation warning.
     if (aliases.some(x => !!x)) {
```

Comment:
This warning should also be updated. Maybe it should be a generic warning? Not sure. I have a PR that will also need to do this.